public/js/graph/parse_graph.js

```diff
@@ -45,7 +45,9 @@ function buildGraph() {
     });
 
     $('.path').each(function () {
-        makeEdge(window[$(this).attr('source-node')], window[$(this).attr('target-node')], $(this).attr('id'));
+        makeEdge(window[$(this).attr('source-node').replace(',', '')],
```

Comment:
I added this in, but I think it can be reverted, as the change is now made on the server.